pages/common/clido.md

```diff
@@ -0,0 +1,36 @@
+# clido
+
+> Save-state TODO app for the terminal.
+> More information: <https://codeberg.org/Oglo12/clido/wiki>.
+
+- Create a list:
+
+`clido --new {{name}}`
+
+- Load a list:
+
+`clido --load {{name}}`
+
+- Delete a list:
+
+`clido --remove {{name}}`
+
+- List all lists:
+
+`clido --lists`
+
+- Toggle autowrite:
+
+`clido toggle-autowrite`
+
+- Open a list in a text editor:
+
+`clido edit {{text_editor}}`
+
+- Show the version:
+
+`clido -v`
+
+- Show a list of commands:
+
+`clido -h`
```

Comment:
Thanks for your contribution and welcome to tldr. I have a minor suggestion for this page.

> `Display version:` and `Display help:` are the generic descriptions we use across most of our pages.

> Also, if long option `--version` and `--help` is supported feel free to update it to be consistent with other examples.

```suggestion
- Display version:

`clido -v`

- Display help:

`clido -h`
```